roles/common/files/uug_ansible_wrapper.py

```diff
@@ -388,6 +388,14 @@ def on_run_clicked(self, _):
         if not validate_branch():
             invalid_branch(self)
             return
+        
+        if (get_distro_release_name() > USER_CONFIG['git_branch']) and (USER_CONFIG['git_url'] == "https://github.com/jmunixusers/cs-vm-build") and (USER_CONFIG['git_branch'] != "master"):
```

Comment:
Can the URL be moved to a global constant called `DEFAULT_GIT_REMOTE` or something similar?